### DISCUSSION TITLE
Convolution on Target

### DISCUSSION THREAD

**Discussion by kathariyabiren** [mit-han-lab/torchsparse#187] [Q&A] [answered]
Mikowski Engine with "MinkowskiConvolutionTranspose" allows convolution to perform on the target. Is there a way to achieve the same with TorchSparse?

**Comment by zhijian-liu** [comment, marked as answer]
This is not yet supported. We will work on this. Thanks for your interest!

**Reply by kathariyabiren** [reply]
I have recently started using TorchSparse more for my research and I have been enjoying it so far. Thank you for your contribution.

**Reply by zhijian-liu** [reply]
Thanks for letting us know! Really glad to hear that TorchSparse is useful in your research. Feel free to reach out if you have any questions.
